WordPress/Resources/release_notes.txt

```diff
@@ -1,3 +0,0 @@
-The “No media matching your search” message in the Media Picker now appears in the right spot. Home, sweet home.
-
-Some users were getting stuck on the “What would you like to focus on first?” screen after logging in, but no more—focus away, friends.
```

Comment:
Notice there are no entries here. I'll raise this internally, but it shouldn't be surprising given the shift in focus towards Jetpack. 

We can get editorialized placeholders notes done anyways.